tox.ini

```diff
@@ -28,7 +28,7 @@ commands =
 
 [testenv:typing]
 deps =
-    mypy
+    mypy<1.1.1
```

Comment:
looking forward for a fix, and presumably already fixed in https://github.com/pydantic/pydantic/pull/5111 (could you check may be), should we just
```suggestion
    mypy!=1.1.1
```
?

Reply:
`<` is slightly better, in the case where mypy releases a new version before pydantic does.

Reply:
Ok, let's proceed but later change when know which needed or to be avoided